extensions/positron-python/python_files/posit/positron/variables_comm.py

```diff
@@ -183,6 +183,9 @@ class VariablesBackendRequest(str, enum.Enum):
     # Request a viewer for a variable
     View = "view"
 
+    # Summarize data
+    SummarizeData = "summarize_data"
+
```

Comment:
We'll have to modify the OpenRPC specs so that the changes in this file can be auto-generated